pkg/action/executor/k8s_controller.go

```diff
@@ -184,10 +189,11 @@ func (c *parentController) update(updatedSpec *k8sControllerSpec) error {
 		}
 
 		if utilfeature.DefaultFeatureGate.Enabled(features.AllowIncreaseNsQuota4Resizing) {
+			start := time.Now()
```

Comment:
The start time should be before we start the parent controller update, isnt it?

Reply:
Yes, I move it in front of the parent controller update